src/app/pages/wallets/wallet-details/pages-wallet-details-item.model.ts

```diff
@@ -0,0 +1,15 @@
+import { IWalletTransactionApiResponse } from 'src/app/domains/wallets/domains.wallets.types';
+
+export class WalletsDetailsTransaction {
+  public readonly id: number | null;
+  public readonly date: Date;
+  public readonly description: string;
+  public readonly amount: number;
+
+  public constructor(data: IWalletTransactionApiResponse) {
+    this.id = data.id === 0 ? null : data.id;
```

Comment:
IDs will start from 1 so it is safe to go with `this.id = data.id ?? null`. Please change this also for `WalletsManagementItem`